Makefile

```diff
@@ -152,7 +152,7 @@ ifdef SAUCE_USER_NAME
 	@echo 'Running web component tests remotely on Sauce Labs...'
 	@npm run test-sauce --silent -- --sauce-tunnel-id \"$(TRAVIS_JOB_NUMBER)\" --sauce-username $(SAUCE_USER_NAME) --sauce-access-key $(SAUCE_ACCESS_KEY)
 else
-	@npm run test -- --local chrome
+	$(MAKE) -e system-test-local
```

Comment:
I think this should not go under this target. This target is meant for running unit tests and wil take place before any dist packaging is done within `make all`. System test should be at the end of the entire process.